run.py

```diff
@@ -153,30 +156,28 @@ def register_node():
 
     logger.info("Registered to server")
     server_response = response.json()["data"]
-    # read worker id
-    if api_key.WORKER_ID in server_response:
-        worker_id = server_response.get(api_key.WORKER_ID)
+    
+    # set heartbeat interval
+    if api_key.HEARTBEAT in server_response:
+        heartbeat_interval = server_response[api_key.HEARTBEAT]
     else:
-        logger.critical(
-            "Bad server response on registration. Missing argument '{}'.".format(
-                api_key.WORKER_ID
-            )
-        )
-        worker_running = False
-        heartbeat_running = False
-        exit(-1)
+        logger.info("Server response did not include heartbeat, using default {}".format(heartbeat_interval))
 
 
 if __name__ == "__main__":
     # check valid usage
-    if len(sys.argv) != 2:
+    if len(sys.argv) != 3:
         print_usage()
         exit(-1)
 
     signal.signal(signal.SIGINT, signal_handler)
 
+    token = sys.argv[1]
```

Comment:
This is probably on me when I initially wrote this. This PR looks good but could we shift to using python's [arg parser](https://docs.python.org/3.5/library/argparse.html)? This will make it easier to see how arguments to enter (using the `- h` flag) and does auto checks for validity of arguments passed.

Reply:
Fixed